### PR TITLE
Disable img lazyload if Incito is lazyloaded

### PR DESCRIPTION
**Pull request by jrlarano**

### CODE REVIEW

File: lib/incito-browser/incito.ts [hunk 1]
```diff
@@ -720,6 +725,8 @@ export default class Incito extends MicroEvent<{
             canLazyload !== undefined
                 ? canLazyload
                 : 'IntersectionObserver' in window;
+        this.enableLazyLoading =
+            enableLazyLoading !== undefined ? enableLazyLoading : false;
```
Comment (klarstrup) [inline lib/incito-browser/incito.ts]: Check if it is a boolean

**Review by klarstrup — CHANGES_REQUESTED** [review]
If I understand the purpose of this: I would remove the options and make this the default behavior specifically for incito embed loads ie. at the `this.renderHtml(res)` call in the embed loading code

File: lib/incito-browser/incito.ts [hunk 2]
```diff
@@ -266,7 +266,7 @@ function renderView(view, canLazyload: boolean) {
             const src = String(new URL(view.src));
 
             if (isDefinedStr(view.src)) {
-                if (canLazyload) {
+                if (canLazyload && !enableLazyLoading) {
```
Comment (mortenbo) [inline lib/incito-browser/incito.ts]: The naming is confusing. You would think enableLazyLoading would enable lazyloading of images. But the enableLazyLoading relates to the Incito

File: lib/incito-browser/incito.ts [hunk 3]
```diff
@@ -866,6 +866,7 @@ export default class Incito extends MicroEvent<{
                     if (res.status === 200) return res.json();
                 })
                 .then((res) => {
+                    this.canLazyload = false;
```
Comment (klarstrup) [inline lib/incito-browser/incito.ts:869]: Not the solution I had in mind, but I can see the appeal 😁

**Review by klarstrup — APPROVED** [review]

**Pull request merged**